indexer/src/test/resources/aa/sample-arba.xml

```diff
@@ -111,7 +113,7 @@
                         <value>IPR032270</value>
                     </condition>
                     <condition type="taxon">
-                        <value cvId="7711">Chordata</value>
+                        <value cvId="10239">Viruses</value>
```

Comment:
Can you please keep Chordata in the xml file?  
This is a good use case for SuperKingdom, because we can reproduce different values for taxonomy and  superkingdom fields

Reply:
Yeah, i will add value "Chordata" back. But i didn't understand he second part of your statement above. How it will be useful?